tests/TestCase.php

```diff
@@ -13,54 +11,37 @@ public function setUp()
     {
         parent::setUp();
 
-        $this->setUpDatabase($this->app);
+        $this->loadMigrationsFrom(realpath(__DIR__.'/fixtures/migrations'));
```

Comment:
I prefer the migration being run inside `TestCase` instead of a separate migration.